lib/pointing_party_web/templates/card/index.html.leex

```diff
@@ -0,0 +1,79 @@
+<div class="row">
+  <div class="card-container col-10">
+    <%= if !@party_has_started do %>
+      <div class="col-md-4 text-center">
+        <button phx-click="start_party" class="btn-primary">Start the Party!</button>
+      </div>
+    <% else %>
+      <div class="card text-left">
+        <div class="card-header">
+          <h2><%= @current_card.title %></h2>
+        </div>
+        <div class="card-body">
+          <p class="card-text"><%= @current_card.description %></p>
+          <form class="form-group text-left points-container" phx-submit="vote">
+            <div class="form-row align-items-center">
+              <div class="col-2">
+                <label for="story-points">Story Points</label>
+                <select name="points" class="form-control story-points" id="story-points">
+                  <option value="1">1</option>
+                  <option value="2">2</option>
+                  <option value="3">3</option>
+                  <option value="5">5</option>
+                </select>
+              </div>
+            </div>
+
+            <%= if @outcome == "winner" do %>
+              <p>Winner: <%= @results %> Points</p>
+              <button
+                <%= if !@is_driving do %>disabled<% end %>
```

Comment:
What do you think about doing away with the concept of a "driver" for this day of the workshop? I think its more useful/makes more sense when we're building these features with channels, but is less necessary here. This could be one way to simplify this day of the workshop.

Reply:
I was storing a lot of the state in the driver's socket state. I'd need to tweak the code so that we store this state in everyone's socket. This should be pretty easy to do, though.

Reply:
Ah yea that makes sense. I could go either way honestly, but I'm leaning towards no more driver. Seems like that will be just slightly easier/less complex to build out and explain. That way we don't have to worry about which buttons to disable, etc.